docs/3.0.0-beta.x/advanced/customize-admin.md

```diff
@@ -33,6 +33,20 @@ The panel will be available through [http://localhost:1337/dashboard](http://loc
 
 **_Currently not available_**
 
+### Styles
+
+Admin part can be easily found in `./node_modules/strapi-admin/src/`.
```

Comment:
"The AdminUI package source can be easily found in `./node_modules/strapi-admin/src/`.